src/main/java/com/google/maps/OkHttpRequestHandler.java

```diff
@@ -15,13 +15,17 @@
 
 package com.google.maps;
 
+import static com.google.appengine.repackaged.com.google.common.base.StringUtil.JsEscapingMode.JSON;
```

Comment:
This is going to be an issue, we don't want to require appengine code for users who aren't on GAE